### PR TITLE
fix(samples): removed env variables and buckets from creating bq.

### PR DESCRIPTION
**Pull request by dfirova**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-retail/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

File: samples/interactive-tutorials/src/main/java/setup/SetupCleanup.java [hunk 1]
```diff
@@ -22,25 +22,8 @@
 import com.google.api.gax.paging.Page;
 import com.google.api.gax.rpc.NotFoundException;
 import com.google.cloud.ServiceOptions;
-import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.*;
```
Comment (chenlei1216) [inline samples/interactive-tutorials/src/main/java/setup/SetupCleanup.java:25]: Should we explictly add all imports?

**Review by chenlei1216 — APPROVED** [review]

**Review by averikitsch — APPROVED** [review]
Please fix broken tests and add comment to where the file is coming from

File: samples/interactive-tutorials/src/main/java/setup/SetupCleanup.java (hunk 1 above)
Comment (dfirova) [reply samples/interactive-tutorials/src/main/java/setup/SetupCleanup.java]: Done.

File: samples/interactive-tutorials/src/main/java/events/setup/EventsCreateBigQueryTable.java [hunk 2]
```diff
@@ -35,11 +37,9 @@ public static void main(String... args) throws IOException {
     String validEventsTable = "events";
     String invalidEventsTable = "events_some_invalid";
     String eventsSchemaFilePath = "src/main/resources/events_schema.json";
-    String validEventsSourceFile =
-        String.format("gs://%s/user_events.json", EventsCreateGcsBucket.getBucketName());
+    String validEventsSourceFile = ProductsCreateBigqueryTable.class.getResource("/user_events.json").getPath();
```
[the remark this answers is not in the document]
Comment (dfirova) [reply samples/interactive-tutorials/src/main/java/events/setup/EventsCreateBigQueryTable.java]: This file is in resources folder.

[the remark this answers is not in the document]
Comment (averikitsch) [reply samples/interactive-tutorials/src/main/java/events/setup/EventsCreateBigQueryTable.java]: You may wish to add a comment noting the location

[the remark this answers is not in the document]
Comment (dfirova) [reply samples/interactive-tutorials/src/main/java/events/setup/EventsCreateBigQueryTable.java]: Done

**Pull request merged**